etc/testing/deploy/aws.sh

```diff
@@ -57,7 +63,7 @@ set -x
 case "${OP}" in
   create)
     aws_sh="$(realpath "$(dirname "${0}")/../../deploy/aws.sh")"
-    sudo "${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics
+    sudo "${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics $CLOUDFRONT
```

Comment:
My memory is that passing an empty flag is not the same thing as passing no flag, so if this doesn't work, you may need to rely on a pattern like
```
cmd=("${aws_sh}" --region=${REGION} --zone=${ZONE} --state=${STATE_STORE} --no-metrics)
if [[ -n "${CLOUDFRONT}" ]]; then
  cmd+=("${CLOUDFRONT}")
fi
```

similar to above...